Documentation/OWNERS

```diff
@@ -0,0 +1,4 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+labels:
+  - area/documentation
```

Comment:
The documentation directory is no longer used in the original way. Originally it included https://github.com/etcd-io/website. Now the remaining documentation is more of contributor documentation, using "area/documentation" label for this directory might be misleading.

Reply:
Yes that directory is rather awkward. I think we have a larger decision to make on if we will shift to serving our contributor facing documentation to a new area of `etcd-io/website`.

To be transparent I would be in favor of this. It's something k8s has been doing for a while https://kubernetes.io/docs/contribute/docs.

I don't want to force that decision in this pr though, happy to come back to it later, perhaps as a topic for one of our upcoming community calls.

Reply:
Based on when the linked pr was merged I would guess the unmaintained issue was a reflection of the state the project was in at that point in time around 2022? We are [in a better position now](https://etcd.devstats.cncf.io/d/74/contributions-chart?orgId=1&var-period=m&var-metric=contributions&var-repogroup_name=etcd-io%2Fwebsite&var-country_name=All&var-company_name=All&var-company=all&from=1638270000000&to=now-2d) in my view with more contributors available for website maintenance.

I like the idea of having better testing for documentation, and actually this isn't limited to contributor docs, I think many of the docs pages for our user facing documentation should have automated testing to ensure they are correct. In the ~2 years since that pr you linked was merged I don't believe we added unit tests for any of the files under our `Documentation/` repo. However if we want to do this there is no reason we can't have scheduled prow job for `etcd-io/website` to let us know if something is broken.

Let's shift this discussion to a community meeting sometime and properly explore some pros/cons. At the end of the day I want our documentation to be the best it can be, irrespective of which git repo it is stored in.